### DISCUSSION TITLE
Council list appears wrong for Tunbridge Wells

### DISCUSSION THREAD

**Discussion by pabutterworth** [robbrad/UKBinCollectionData#931] [General]
Hi,

The council list entry for Tunbridge wells shows Lincoln, assume this is an error?

Thanks
Paul

Tunbridge Wells Council

python collect_data.py LincolnCouncil https://lincoln.gov.uk -u XXXXXXXX -p "XXXX XXX"
Additional parameters:

-u - UPRN
-p - postcode
Note: You will need to use [FindMyAddress](https://www.findmyaddress.co.uk/search) to find the UPRN.

**Comment by dp247** [comment]
Moved to #963 as this is a bug, so not suitable for discussion format
